src/devices/video/upd7220.cpp

```diff
@@ -1531,7 +1545,31 @@ void upd7220_device::write(offs_t offset, uint8_t data)
 
 uint8_t upd7220_device::dack_r()
 {
-	return 0;
+	uint8_t result = 0;
+	switch(m_dma_type) {
```

Comment:
You should be consistent with the rest of the file – this file uses Allman style.